src/Resources/Resources/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 ## Upcoming Release
 * Fix bug where New-AzRoleAssignment could not be called without parameter Scope.
+* Add new cmdlet New-AzDeploymentWhatIf for deployment what-if operation.
```

Comment:
You should update your branch with the latest changes on master - the above changelog entry had already been moved to 1.7.0

Reply:
Copied and updated the change log from master manually...I tried to do a merge from master but it added too much noise to the PR (300+ files) as it is not targeting mater.

Reply:
Sorry I didn't notice you are targeting the preview branch...
You probably already know there's an ongoing preivew release for Az.Resource. After that is done (after #10251 is merged), you'll need to update your branch with the preview branch.

Reply:
Thanks for letting me know! I've merged the branch.